include/swift/ClangImporter/ClangImporter.h

```diff
@@ -705,6 +705,9 @@ bool isCxxConstReferenceType(const clang::Type *type);
 /// Determine whether this typedef is a CF type.
 bool isCFTypeDecl(const clang::TypedefNameDecl *Decl);
 
+// Determine whether type is a c++ foreign reference type.
```

Comment:
Minor: we prefix C++ doc comments with `///` to mark a doc comment and make it show up in IDE quick help.